tests/run_all.sh

```diff
@@ -7,7 +7,7 @@ export HOME=/opt/spark/work-dir
 export PATH=$PATH:/opt/spark/bin:/opt/spark/work-dir/.local/bin
 
 echo "Installing tox ..."
-pip3 install -q tox
+pip3 install -q tox-uv
```

Comment:
uv is a much faster pip alternative, very nice for faster local iterations on tests, not much of a difference for ci, is dwarfed by build times